go/cmd/server/logging/filelogstorage_test.go

```diff
@@ -0,0 +1,89 @@
+package logging
+
+import (
+	"path/filepath"
+
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/apache/arrow/go/v8/arrow"
+	"github.com/apache/arrow/go/v8/arrow/array"
+	"github.com/apache/arrow/go/v8/arrow/memory"
+	"github.com/apache/arrow/go/v8/parquet/file"
+	"github.com/apache/arrow/go/v8/parquet/pqarrow"
+	"github.com/feast-dev/feast/go/internal/test"
+	"github.com/feast-dev/feast/go/protos/feast/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFlushToStorage(t *testing.T) {
+	table, err := GenerateLogsAndConvertToArrowTable()
+	defer table.Release()
+	assert.Nil(t, err)
+	offlineStoreConfig := map[string]interface{}{
+		"path": ".",
+	}
+	fileStore, err := NewFileOfflineStore("test", offlineStoreConfig)
+	assert.Nil(t, err)
+	err = fileStore.FlushToStorage(array.Table(table))
+	assert.Nil(t, err)
+	logPath, err := filepath.Abs(filepath.Join(".", "log.parquet"))
+	assert.Nil(t, err)
+	w, err := CreateOrOpenLogFile(logPath)
+	assert.Nil(t, err)
+	ctx := context.Background()
+
+	pf, err := file.NewParquetReader(w)
+	assert.Nil(t, err)
+
+	reader, err := pqarrow.NewFileReader(pf, pqarrow.ArrowReadProperties{}, memory.DefaultAllocator)
+	assert.Nil(t, err)
+
+	tbl, err := reader.ReadTable(ctx)
+	assert.Nil(t, err)
+	tr := array.NewTableReader(tbl, -1)
+	defer tbl.Release()
+	expected_schema := map[string]arrow.DataType{
+		"driver_id": arrow.PrimitiveTypes.Int64,
+		"int32":     arrow.PrimitiveTypes.Int32,
+		"double":    arrow.PrimitiveTypes.Float64,
+		"int64":     arrow.PrimitiveTypes.Int64,
+		"float32":   arrow.PrimitiveTypes.Float32,
+	}
+
+	expected_columns := map[string]*types.RepeatedValue{
```

Comment:
ideally we should reuse here data generated above. Because in order to change any value I need to change it in several places (should be one)

Reply:
Done.